.github/workflows/test_templated_agent.yaml

```
@@ -0,0 +1,133 @@
+          echo "[]" > "$TEMP_JSON_FILE"
+
+          echo "${{ steps.changed_files.outputs.all_changed_files }}" | \
+          grep -o 'python/agents/[^/]*' | \
+          sort -u | \
+          while read -r dir; do
+            if [ -f "$dir/.template/templateconfig.yaml" ]; then
+              deployment_targets=$(yq eval '.settings.deployment_targets[]' "$dir/.template/templateconfig.yaml" 2>/dev/null || echo "")
+              if [ -z "$deployment_targets" ]; then
+                deployment_targets="agent_engine cloud_run"
+              fi
+              
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)